readymade.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |spec|
   spec.description   = 'Set of base classes for ABDI architecture'
   spec.homepage      = 'https://github.com/OrestF/readymade'
   spec.license       = 'MIT'
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.3.0')
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.7.0')
```

Comment:
The [reason]([step](https://github.com/dmrAnderson/readymade/actions/runs/10126025428/job/28002191424#step:3:38)) why I increased the ruby version